Views/mobile.css

```diff
@@ -0,0 +1,115 @@
+
```

Comment:
We can add the licence header here though.

Reply:
Done.